chidioguejiofor/airtech-api
# #167104120: Adds missing patch and delete routes

#### What does this PR achieve?
- adds an implementation of DELETE booking and PATCH flight
- adds a pull request template


#### Any background context?
- None


#### Manual Testing Steps
- Migrate your data to your database via `python manage.py makemigrations && python manage.py migrate`
- Try to DELETE  a flight as an admin via '/api/v1/flights/{fligth-id}'
- You should get a message saying that the flight has been deleted
